tutorials/plot_receptive_field.py

```diff
@@ -307,6 +307,15 @@
 
 ix_best_alpha = np.argmax(scores)
 
+###############################################################################
+# Compare model performance
+# -------------------------
+# Below we visualize the model performance of each regularization method
+# (ridge vs. quadratic) for different levels of alpha. As you can see, the
+# quadratic method performs better in general, because it imposes a smoothness
+# constraint along the time dimension of the coefficients. This matches
```

Comment:
ah ok - I didn't realize you'd implemented both. Is that a hard requirement? Does that assumption hold if, e.g., you had log-spaced features?